config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+Sidekiq.configure_server do |config|
+  config.redis = { url: Settings.redis.sidekiq.uri, size: 15 }
+end
+
+Sidekiq.configure_client do |config|
+  config.redis = { url: Settings.redis.sidekiq.uri, size: 2 }
```

Comment:
using the calculator, this seems to be set to 5 but per our conversation only 2 for client might be enough.